clickhouse_search/backend/engines.py

```diff
@@ -5,7 +5,6 @@
 def _no_validate(value, name):
     return value
 
-
```

Comment:
not very important but I think it is the convention to have 2 blank lines between top level functions